src/plugins/api/api.ts

```diff
@@ -77,6 +94,16 @@ export class Api extends ApiBase {
     return false;
   }
 
+  /**
+   * Pings the server (via {@linkcode getGameTitleStats}) and updates {@linkcode _isConnected} accordingly.
+   */
+  async ping() {
+    if (this.isLocal) {
+      const titleStats = await this.getGameTitleStats();
+      this._isConnected = !!titleStats;
+      console.log("isLocalServerConnected:", this.isConnected);
+    }
```

Comment:
```suggestion
    const titleStats = await this.getGameTitleStats();
    this._isConnected = !!titleStats;
    console.log("isLocalServerConnected:", this.isConnected);
```
(Since `_isConnected` has no default value, we also need to update its value when `isLocal` is false)